docs/api/rest_api/rest_api_reference/input/ibexa-ai.raml

```diff
@@ -1,5 +1,5 @@
-displayName: AI Actions
-/execute/{actionConfiguration}:
+displayName: AI actions
+/actions/execute/{actionConfiguration}:
```

Comment:
The proper URL in the newest version is:
/ai/actions/{actionConfiguration}/execute